dist/well-known.js

```diff
@@ -115,6 +116,54 @@ return Promise.all([
       return data;
     });
   }),
+  // FedCM
+  parseResponse('/.well-known/web-identity', r => {
+    return r.json().then(data => {
+        let result = {
+            provider_urls: Array.isArray(data.provider_urls) && data.provider_urls.length > 0 ? data.provider_urls : [],
+            accounts_endpoint: data.accounts_endpoint || null,
+            login_url: data.login_url || null,
+            valid: Array.isArray(data.provider_urls) && data.provider_urls.length > 0
```

Comment:
`valid` is based on `provided_urls` so don't need to log it and can analyze it when querying the data:
```suggestion
            provider_urls: Array.isArray(data.provider_urls) && data.provider_urls.length > 0 ? data.provider_urls : [],
            accounts_endpoint: data.accounts_endpoint || null,
            login_url: data.login_url || null
```